py/dml/dmlparse.py

```diff
@@ -1144,6 +1144,9 @@ def sizespec_empty(t):
 @prod
 def offsetspec(t):
     'offsetspec : AT expression'
+    if (t[2].kind == 'undefined' and site(t).dml_version() == (1, 2)
+        and logging.show_porting):
+        report(PUNDEFOFFS(site(t, 2)))
```

Comment:
Instead of doing it here, we could check it in `reginfo.explode_register()` to cover `param offset = undefined` for a small additional complexity cost. Yes or no?

Reply:
that's nice, in particular if you can hide it in an existing version==1.2 clause. But not strictly needed.